src/main/kotlin/app/extractors/ExtractorInterface.kt

```diff
@@ -6,18 +6,23 @@ package app.extractors
 
 import app.BuildConfig
 import app.Logger
+import app.ModelsProtos.ModelsVersions
```

Comment:
Please, implement wrapper for ModelVersionGroup / ModelVersion protobuf messages like: [Classifier](https://github.com/sourcerer-io/sourcerer-app/blob/develop/src/main/kotlin/app/extractors/Classifier.kt) or [CommitGroup](https://github.com/sourcerer-io/sourcerer-app/blob/develop/src/main/kotlin/app/model/CommitGroup.kt). ExtractorInterface should not deal with encoding, it's already doing too much.